frontend/src/pages/instance/Editor/index.vue

```diff
@@ -55,6 +55,8 @@
             </ff-page>
         </section>
 
+        <ConfirmInstanceDeleteDialog ref="confirmInstanceDeleteDialog" :instance="instance" @confirm="onInstanceDelete" />
+
```

Comment:
found a missing feature while working on this, we were unable to delete instances whilst using the immersive editor because the delete dialog was missing